tasks/__init__.py

```diff
@@ -0,0 +1,6 @@
+# -*- coding: utf-8 -*-
+"""Consolidates invoke tasks and utilities
+
+"""
+from .tasks import *  # noqa
```

Comment:
We wouldn't import tasks in the application code, but `invoke` must be importing the tasks code; having two conflicting modules to import `tasks.py` and `tasks/__init__.py` is hard to reason about.

It's not necessarily a bad thing to have code in `__init__.py`. For readability, we might break up `tasks/__init__.py`  into multiple files (and then import those modules in `__init__.py`), but that doesn't need to happen now.